src/PrestaShopBundle/Resources/views/Admin/Sell/Customer/Blocks/View/viewed_products.html.twig

```diff
@@ -41,8 +41,8 @@
         <tbody>
           {% for product in customerInformation.productsInformation.viewedProductsInformation %}
             <tr>
-              <td>{{ product.productId }}</td>
-              <td>
+              <td class="customer-viewed-product-id">{{ product.productId }}</td>
+              <td class="customer-viewed-product-name">
                 <a href="{{ product.productUrl }}">
```

Comment:
I think we should add some consistency on our integration workflow, I would love to have a class on every childs like :

```suggestion
                <a href="{{ product.productUrl }}" class="customer-viewed-product-link">
```